src/context/yaml/handlers/clients.js

```diff
@@ -22,7 +22,7 @@ async function parse(context) {
           const htmlFileName = path.join(clientsFolder, client.custom_login_page);
 
           if (isFile(htmlFileName)) {
-            client.custom_login_page = loadFile(htmlFileName);
+            client.custom_login_page = context.loadFile(htmlFileName);
```

Comment:
wheres `loadfile` set on `context`? Was it already there?

Reply:
Yep: https://github.com/auth0/auth0-deploy-cli/blob/9b44071c5f01287c26a94a189ad15e1148e5fd3c/src/context/yaml/index.js#L36